SAKPaaS/src/assets/i18n/de.json

```diff
@@ -11,7 +11,8 @@
   },
   "footer": {
     "help": "Hilfe",
-    "legal": "Rechtliches",
+    "impressum": "Impressum",
+    "privacy-policy": "Privacy Policy",
```

Comment:
Hier sollte "Datenschutz" oder so stehen

Reply:
@elsholz Wenn du das fixt ist ready to merge